charts/eks-anywhere-packages/templates/package.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if not .Values.workloadOnly }}
+{{-  $namespace := printf "%s-%s" "eksa-packages" .Values.clusterName -}}
+{{- if not (lookup "packages.eks.amazonaws.com/v1alpha1" "Package" "eksa-packages-main" "eks-anywhere-packages") }}
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  annotations:
+    "helm.sh/resource-policy": keep
+  name: eks-anywhere-packages
```

Comment:
So moving the namespace to the CRDs would make it so upgrading itself wouldn't fail right?